### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by sfitz42**

### CODE REVIEW

File: src/services/ai/chains/generate.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import {
+  RunnableSequence,
+  RunnablePassthrough
+} from '@langchain/core/runnables'
+import { formatDocumentsAsString } from 'langchain/util/document'
+import { JsonOutputParser } from '@langchain/core/output_parsers'
+
+const generatePrompt = `
+[INST]
+  You are a Gov UK DEFRA AI Assistant, whose job it is to retrieve and summarise information regarding available grants for farmers and land agents.
```
Comment (sebhoron) [inline src/services/ai/chains/generate.js:14]: Remove 'it' from the 'job it is'

File: src/services/ai/chains/generate.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  RunnableSequence,
+  RunnablePassthrough
+} from '@langchain/core/runnables'
+import { formatDocumentsAsString } from 'langchain/util/document'
+import { JsonOutputParser } from '@langchain/core/output_parsers'
+
+const generatePrompt = `
+[INST]
+  You are a Gov UK DEFRA AI Assistant, whose job it is to retrieve and summarise information regarding available grants for farmers and land agents. 
+  [CONTEXT] will be provided to you with two constituent parts; an identifier and the content. The identifier will be at the start of the document, within a set of parentheses in the following format:
```
Comment (sebhoron) [inline src/services/ai/chains/generate.js:15]: Should there be ':' after parts instead of ';'?

**Review by sebhoron — COMMENTED** [review]
Just left a few comments

**Comment by sfitz42** [comment]
@sebhoron - Agreed on both counts but we're going to keep it the same atm as it's just a straight forward migration.

Just so we have a direct comparison with the previous versions.

**Review by sebhoron — APPROVED** [review]
LGTM

**Pull request merged**